### DISCUSSION TITLE
Lab 1 Part 1 Feedback

### DISCUSSION THREAD

**Discussion by shivanidoshi26** [shivanidoshi26/Interactive-Lab-Hub#5] [General]
Add your Lab 1 feedback to this thread please!

**Comment by Rpoddar1953** [comment]
From the video I saw that the device gives you a red if the food is not good to eat/ripe and green if it is. The video showed that a more ripe tomato has a different color, but I guess I was wondering how clear it is that a riper fruit has a darker/different color, cause I couldn't really tell form the video. I was also wondering if there is a difference in color between a food not being ripe and something not being edible such as the wrapper on the candy.

**Reply by snlee159** [reply]
Copied from slack:

- the stuff I mentioned before: like maybe it could go on the grocery cart itself
- the device doesn’t have any buttons to say you’re ready to capture stuff. A “test freshness” button could be useful
- Maybe also some words to express what each color means. Like a shopper might not automatically know what the different shades of green imply
- More a feedback on tinkerbelle than your design but the different greens definitely aren’t showing up in the videos and so could be harder to interpret
- Your actress is top notch. I hope she got paid the big bucks

**Comment by nagrawal44** [comment]
Great video, easy to understand what's happening. setting is a grocery store and the purpose of the device is to know whether its ripe or not ripe. in reality with this device, there will have to be some type of key on what each color indicates. Is red always no good, and green is good? Are you using other colors for different types of products like blue indicates that the frozen item is frozen well? Also you can't unwrap items in a grocery store, so your light device will have to work through the wrapper. Its a useful product bc I would love to know if my avocados are ready, and I agree that sound is not needed.
